change/@fluentui-react-charting-cfa801e0-6573-413b-bb65-4c94f5e8ddbe.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Sankey chart changes for the new design",
+  "packageName": "@fluentui/react-charting",
+  "email": "ankityadav@microsoft.com",
+  "dependentChangeType": "patch"
+}
```

Comment:
What all accessibility tests have you performed.

Reply:
I need to figure out the logic for getting the order in which screen reader read the sankey chart., apart from that screen reading , focus on elements with tab is done .